### PR TITLE
parser/executor: admin show/set bdr role

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb/issues/48519

Problem Summary:

### What is changed and how it works?

- admin show bdr role
- admin set bdr role none/primary/secondary/local_only

### Check List

Tests 

- [x] Unit test
- [x] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: tests/integrationtest/t/executor/executor.test [hunk 1]
```diff
@@ -2537,3 +2537,14 @@ from ssci right join csci on (ssci.customer_sk=csci.customer_sk
...
+admin show bdr role;
+admin set bdr role primary;
+admin show bdr role;
+admin set bdr role secondary;
+admin show bdr role;
+admin set bdr role local_only;
+admin show bdr role;
+--error 1064
+admin set bdr role test_err;
+admin show bdr role;
```
Comment (tangenta) [inline tests/integrationtest/t/executor/executor.test:2550]:
```suggestion
admin show bdr role;

```

**Comment by okJiang** [comment]
ping @tangenta @GMHDBJD

**Review by tangenta — APPROVED** [review]

File: pkg/parser/parser.y [hunk 2]
```diff
@@ -6525,6 +6529,7 @@ UnReservedKeyword:
 |	"AFTER"
 |	"ALWAYS"
 |	"AVG"
+|   "BDR"
```
Comment (ywqzzy) [inline pkg/parser/parser.y:6532]: more indent？

File: pkg/parser/parser.y [hunk 3]
```diff
@@ -6582,6 +6587,7 @@ UnReservedKeyword:
 |	"INSERT_METHOD"
 |	"LESS"
 |	"LOCAL"
+|   "LOCAL_ONLY"
```
Comment (ywqzzy) [inline pkg/parser/parser.y:6590]: ditto

File: pkg/parser/parser.y (hunk 2 above)
Comment (okJiang) [reply pkg/parser/parser.y]: fixed in https://github.com/pingcap/tidb/pull/48504/commits/bd24d7266455c71f8e43958760ab15a27432eb82

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**